expertsystem/amplitude/_yaml_adapter.py

```diff
@@ -320,6 +320,31 @@ def _extract_intensity_component(definition: Dict[str, Any]) -> Dict[str, Any]:
                 recoil_system["RecoilFinalState"]
             )
             output_dict["RecoilSystem"] = recoil_system
+        if "CanonicalSum" in definition:
+            cano_sum_old = definition["CanonicalSum"]
+            cano_sum_new = dict()
+            cano_sum_new["L"] = cano_sum_old["L"]
+            cano_sum_new["S"] = cano_sum_old["S"]
+            clebsch_gordan_list = [
+                dict(item)  # conversion from OrderedDict
+                for item in _safe_wrap_in_list(cano_sum_old["ClebschGordan"])
+            ]
+            clebsch_gordan_dict = dict()
+            for clebsch_gordan_old in clebsch_gordan_list:
+                type_name = clebsch_gordan_old["Type"]
+                clebsch_gordan_new = {
+                    "J": clebsch_gordan_old["J"],
+                    "M": clebsch_gordan_old["M"],
+                }
+                attributes = {
+                    key[1:]: value
+                    for key, value in clebsch_gordan_old.items()
+                    if key.startswith("@")
+                }
+                clebsch_gordan_new.update(attributes)
+                clebsch_gordan_dict[type_name] = clebsch_gordan_new
+            cano_sum_new["ClebschGordan"] = clebsch_gordan_dict
+            output_dict["CanonicalSum"] = cano_sum_new
```

Comment:
So I ran the test script and checked the output:
```yml
CanonicalSum:
    L: 2
    S: 1.0
    ClebschGordan:
        LS:
            J: 1.0
            M: -1.0
            j1: 2.0
            m1: 0.0
            j2: 1.0
            m2: -1.0
        s2s3:
            J: 1.0
            M: -1.0
            j1: 0.0
            m1: 0.0
            j2: 1.0
            m2: -1.0      
```
There is redundant information. I would propose this:

```yml
Canonical:
    LS:
        ClebschGordan:
            J: 1.0
            M: -1.0
            j1: 2.0
            m1: 0.0
            j2: 1.0
            m2: -1.0
    s2s3:
        ClebschGordan:
            J: 1.0
            M: -1.0
            j1: 0.0
            m1: 0.0
            j2: 1.0
            m2: -1.0      
```
I used the ClebschGordan label, because I want this to be like a mathematical command.

Reply:
Alright, I'll implement that, won't be hard. I trust that `L` and `S` are redundant here?

Reply:
Yeah, the j1 and j2 in the first block are L and S and couple to J. Also in the second block the J value is equal to S.

Reply:
ed2347f9494d933dbbbe1bdc66ad13a8e14dc270